core/domain/suggestion_registry.py

```diff
@@ -654,6 +724,48 @@ def _get_skill_difficulty(self):
         """Returns the suggestion's skill difficulty."""
         return self.change.skill_difficulty
 
+    def get_all_html_content_strings(self):
+        """Gets all html content strings used in this suggestion.
+
+        Returns:
+            list(str). The list of html content strings.
+        """
+        html_string_list = []
+        interaction_object = (
+            state_domain.InteractionInstance.from_dict(
+                self.change.question_dict['question_state_data'][
+                    'interaction']))
+        html_string_list = (
+            html_string_list +
+            interaction_object.get_all_html_content_strings())
+        written_translations_dict = (
+            self.change.question_dict['question_state_data'][
+                'written_translations'])
+        written_translations_object = (
```

Comment:
By default, a variable name with _dict as a suffix is considered as object so remove the suffix here

Reply:
done